assert/assertions.go

```diff
@@ -1874,6 +1874,9 @@ type CollectT struct {
 	errors []error
 }
 
+// Helper is like [testing.T.Helper] but does nothing.
+func (CollectT) Helper() {}
```

Comment:
Should the newly merged `captureTestingT` also get a `Helper()` method? (Or, should `captureTestingT` be replaced with `CollectT`?)